crates/fraud_proofs/global_merkle_root/storage/src/merkle/smt.rs

```diff
@@ -0,0 +1,60 @@
+use crate::{
+    column::Column,
+    merkle::MerklizedTableColumn,
+};
+use fuel_core_storage::{
+    blueprint::plain::Plain,
+    codec::{
+        postcard::Postcard,
+        primitive::Primitive,
+        raw::Raw,
+    },
+    structured_storage::TableWithBlueprint,
+    tables::merkle::SparseMerkleMetadata,
+    Mappable,
+};
+use fuel_core_types::fuel_merkle::sparse;
+
+/// The table of SMT data for Contract state.
+pub struct MerkleData<Table>(core::marker::PhantomData<Table>);
+
+impl<Table> Mappable for MerkleData<Table> {
+    type Key = [u8; 32];
+    type OwnedKey = Self::Key;
+    type Value = sparse::Primitive;
+    type OwnedValue = Self::Value;
+}
+
+impl<Table> TableWithBlueprint for MerkleData<Table>
+where
+    Table: MerklizedTableColumn,
```

Comment:
nit, but I believe the correct spelling is `Merkleized`
```suggestion
    Table: MerkleizedTableColumn,
```

Reply:
Sure, I'll update.

Reply:
Done in https://github.com/FuelLabs/fuel-core/pull/2553/commits/6b0d8196f1fa215fba6007e83f3edb5622f575bc